jenkins/Jenkinsfile

```diff
@@ -259,32 +259,26 @@ pipeline {
 
                         if (shouldUpdate) {
                             for (String image: images) {
+                                def varName = image.replace("-","_")
+
                                 // The version could be empty if the image from terra-docker is not in a leo conf yet.
                                 if (image in versionMap.keySet() && !versionMap[image].isEmpty()) {
                                     version = versionMap[image]
 
                                     if (GENERATE_DATAPROC_IMAGE == "true" && dataprocImages.contains(image)) {
-                                        if (image != 'terra-jupyter-aou') {
-                                            println("Using version $version for '$image' in this build of custom Dataproc images")
-                                            sh """
-                                              sed -i "s/$image.*/$image:$version\\"/" jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh
-                                              """
-                                        } else {
-                                            println("Not updating version for image $image")
-                                        }
-
+                                        println("Using version $version for '$image' in this build of custom Dataproc images")
+                                        // The below sed command does not update *_old variables, which must be manually updated
+                                        sh """
+                                            sed -i "s/^$varName=\\(.*\\)$image.*\$/$varName=\\1$image:$version\\"/" jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh
```

Comment:
This will update all known images to the latest version, but leave the `_old` images alone.

Reply:
btw, I think aou is using `2.0.1` in prod now (noticed this when looking into an AoU issue previously)..just want to make sure `2.0.1` is old now

Reply:
ohh good catch! Yeah seems so: https://github.com/all-of-us/workbench/blob/5af159fb974b7a542c2eb30bfbe5b41053368523/api/config/config_prod.json#L11

Will update `_old` to be `2.0.1` then.

Reply:
This is done now - regenerated the images and updated this PR.